consts.js

```diff
@@ -0,0 +1,4 @@
+const SDAROT_USERNAME_KEY = 'sdarotUsername';
```

Comment:
You can add 'export' before the const declaration so you will be able to import like this - `import {SDAROT_USERNAME_KEY } from './consts.js`

Reply:
I tried that, maybe we can get it to work, let me check